src/applications/combined-debt-portal/combined/components/Balances.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import head from 'lodash/head';
+import moment from 'moment';
+import { useSelector } from 'react-redux';
+import BalanceCard from './BalanceCard';
+import ZeroBalanceCard from './ZeroBalanceCard';
+import AlertCard from './AlertCard';
+import ComboAlerts from './ComboAlerts';
+
+export const IS_DEBT = true;
```

Comment:
THOUGHT: This feels really weird to me. I see how you got to this. I have done stuff like this in the past and usually had to refactor it out later.  My main gripes would be that booleans like this tend to hide the negative case and restricts the app to only 2 states. I know there are only two states here today, so this is mostly food for thought for now. 

Off the top of my head, two different ideas to handle this in more extendable way would be to : 

1. Instead of using a boolean, use a hash that has the app type. It would something like 
```js
const APP_TYPES = {
 DEBT:"DEBT",
 COPAY: "COPAY"
}
// later in component
const AlertCard = ({appType})=> {
 // this could now be expanded to support other types of apps
 const displayThis = appType === APP_TYPE.DEBT ? "A" : "B";
 return <p> {displayThis}</p>
}
// Even more later in parent component.
<AlertCard appType={APP_TYPE.DEBT}/>

```

2. Do the component namespacing thing you are doing with the Alerts and have something like 
``` jsx
const BaseBalance = ({header, linkText, foobar}) => {
 return <p>{header}</p>
} 
const DebtBalance = () => {
const value = someCustomLogic
return <BaseBalance header="debt stuff" foobar=value/>
}
const CopayBalance = () => {
const value = someOtherCustomLogic
 return <BaseBalance header="copay stuff" foobar=value/>
}

export {
  DebtBalance, 
  CopayBalance, 
}
```

Each way works and has its pros and cons. Since I know this is only a 2 app application, I don' think this is a must-do, more of a, be aware that this is a current limitation (which is not as bad as it sounds).